new-runtimepoc/Pester.Runtime.psm1

```diff
@@ -856,6 +856,7 @@ function New-TestObject {
         Executed          = $false
         ExecutedAt        = $null
         Passed            = $false
+        Result            = $null
```

Comment:
Where is this used and what it means? I can't find any usage, and since the runtime only supports passed / skipped / failed, this is not needed (yet) because all the states can be determined from the data that are on the object (eg. -not ShouldRun = skipped, ShouldRun -and Executed -and Passed = success).

Reply:
Will remove. I think it is a leftover from when I was trying some other things

Reply:
Done